### PR TITLE
Guided Tours: Colon in header and unused strings

### PR DESCRIPTION
**Pull request by Kostelano**
Pull Request for Issue # .

### Summary of Changes
1. In the title of the page above the toolbar, a colon is usually used, not a dash.

Screenshot_1

2. Unused localization strings that were probably used in the early stages of writing the component's code.

### CODE REVIEW

File: administrator/components/com_guidedtours/src/View/Step/HtmlView.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ protected function addToolbar()
 
         $canDo = ContentHelper::getActions('com_guidedtours');
 
-        ToolbarHelper::title(Text::_('COM_GUIDEDTOURS') . ' - ' . ($isNew ? Text::_('COM_GUIDEDTOURS_MANAGER_STEP_NEW') : Text::_('COM_GUIDEDTOURS_MANAGER_STEP_EDIT')), 'map-signs');
+        ToolbarHelper::title(Text::_('COM_GUIDEDTOURS') . ': ' . ($isNew ? Text::_('COM_GUIDEDTOURS_MANAGER_STEP_NEW') : Text::_('COM_GUIDEDTOURS_MANAGER_STEP_EDIT')), 'map-signs');
```
Comment (Quy) [inline administrator/components/com_guidedtours/src/View/Step/HtmlView.php]: This should be done in the language string. For example:

```
COM_CONTENT_PAGE_ADD_ARTICLE="Articles: New"
COM_CONTENT_PAGE_EDIT_ARTICLE="Articles: Edit"
COM_CONTENT_PAGE_VIEW_ARTICLE="Articles: View"
```

Comment (Kostelano) [reply administrator/components/com_guidedtours/src/View/Step/HtmlView.php]: @Quy Excuse me, but what is the fundamental difference?

Comment (Fedik) [reply administrator/components/com_guidedtours/src/View/Step/HtmlView.php]: @Quy is right. That how it done in other places. Example Menu https://github.com/joomla/joomla-cms/blob/f05a062d9dab659d82f313c2dcf2fda268b4ac30/administrator/components/com_menus/src/View/Item/HtmlView.php#L139

Comment (Kostelano) [reply administrator/components/com_guidedtours/src/View/Step/HtmlView.php]: Done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 13c83c01bd51681c408e2f59a78a628b4c3ddd6b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40212">issues.joomla.org/tracker/joomla-cms/40212</a>.</sub>

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 13c83c01bd51681c408e2f59a78a628b4c3ddd6b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40212">issues.joomla.org/tracker/joomla-cms/40212</a>.</sub>

**Comment by Fedik** [comment]
r2c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40212">issues.joomla.org/tracker/joomla-cms/40212</a>.</sub>

**Comment by obuisard** [comment]
Thank you Konstantin @Kostelano for the PR!

**Pull request merged**